src/util-html-to-md.ts

```diff
@@ -6,6 +6,7 @@ export const toMdNode = (node: RemarkNode): RemarkNode | null => {
   const value = node.value
   const parsed = parse5.parseFragment(value)
   const imgNode = parsed.childNodes.find(node => node.tagName === 'img')
+  if (!imgNode) {return node}
```

Comment:
I think we want to return `null` here (similar to this [line below](https://github.com/d4rekanguok/gatsby-remark-images-anywhere/blob/581cf5766596635c8ca8d9470a77b9237f7a5686/src/util-html-to-md.ts#L16)), since we want to filter out any html nodes that's not an `img` tag

Reply:
Oop, okay! Does that require a new pull request?

Reply:
nice, you got it!